corehq/apps/sms/models.py

```diff
@@ -355,7 +355,7 @@ class Meta(object):
     def get_queued_sms(cls):
         return cls.objects.filter(
             datetime_to_process__lte=datetime.utcnow(),
-        )
+        ).order_by('datetime_to_process')
```

Comment:
the only use case of this method is when we need to run queued SMSes, so it sounded okay to put this here itself instead of where it is called. Plus also felt right for the method itself to return them in that order.

Reply:
Agreed, I think this method being named **queued** sms makes adding an order within this function reasonable.